ignite/templates/module/create/files/base/x/{{moduleName}}/keeper/keeper.go.plush

```diff
@@ -82,65 +71,3 @@ func NewKeeper(
 func (k Keeper) GetAuthority() []byte {
 	return k.authority
 }
-
```

Comment:
Is this not necessary anymore?

Reply:
Nope capability doesn't exist in IBC v10